### DISCUSSION TITLE
Wireclick for column

### DISCUSSION THREAD

**Discussion by jay2success** [rappasoft/laravel-livewire-tables#743] [Q&A] [answered]
Hi,

Is there "wireclick" feature for column example to LinkColumn?
So the link is not opening link but using wireclick.
It will be more flexible so we can use custom action.
Thanks.

**Comment by rappasoft** [comment, marked as answer]
I would use: https://rappasoft.com/docs/laravel-livewire-tables/v2/datatable/available-methods#content-settrattributes

```php
public function configure(): void
{
  // Takes a callback that gives you the current row and its index
  $this->setTrAttributes(function($row, $index) {
      return [
        'wire:click' => 'myFunction('.$row->id.')',
     ];
  });
}
```

**Reply by jay2success** [reply]
Thanks a lot for your answer, i will try this.
